src/helpers/gpt.ts

```diff
@@ -46,22 +46,12 @@ export const issueSpecExample = `
 `;
 
 export const specCheckTemplate = `
-  You are an AI designed to analyze pull requests in comparison to the issue specification.\n
-  Your utmost priority is to ensure the pull request is in line with the issue spec.\n
-  Detail the specification in as much detail as possible, and analyze the pull request in comparison to this specification.\n
-  Feed your analysis into the boilerplate provided below, changing what you need to.\n
-  Be concise and group your analysis into relevant sections.\n
-
-  Boilerplate: \n
-  ${issueSpecExample}
-  `;
-
-export const speckCheckResponse = `
-Finalize the report by proofreading the report for any errors.
-Improve the report by adding any additional information you deem relevant.
-Remove anything that is not relevant to the report, if you can replace it with something more relevant, do so.
-
-All replies MUST end with "\n<!--- { 'OpenAI': 'specCheck' } --->".
+Using the provided context, ensure you clearly understand the specification of the issue.
```

Comment:
This is a much more streamlined prompt. Thanks!

Reply:
Ah yeah I'm no longer having to check for previous reports, will have it removed.

The responses are throwing a lot of false positives and occasionally are far too anal about the spec so it still needs worked I think. Things like changes to the .gitignore or not being mentioned in the spec for example silly things like that, will try to find a way to have the second call filter these sorts of things.

Reply:
I don't think I see an issue with that. Extraneous changes are not desirable. It's like when assignees add extra dependencies and don't use them in the code. I would ding them in the review for that and ask them to remove it for example.

Reply:
I feel this can be resolved now.